http-server/src/test/java/xtdb/jackson/JsonLdValueOrPersistentHashMapDeserializerTest.java

```diff
@@ -0,0 +1,74 @@
+package xtdb.jackson;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import clojure.java.api.Clojure;
+import clojure.lang.*;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.*;
+import com.fasterxml.jackson.databind.module.SimpleDeserializers;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+class JsonLdValueOrPersistentHashMapDeserializerTest {
+
+    private final ObjectMapper objectMapper;
+    private IFn keywordFn = Clojure.var("clojure.core", "keyword");
+    private IFn setFn = Clojure.var("clojure.core", "set");
+
+    public JsonLdValueOrPersistentHashMapDeserializerTest() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(JsonParser.Feature.ALLOW_SINGLE_QUOTES, true);
```

Comment:
In the tests we have (at least) Java 17 available, which means we have text blocks:
```java
"""
{"@value": "...", "@type": "..."}
"""
```

Would prefer we used this rather than single quotes because it'll be more representative of the real world systems